### DISCUSSION TITLE
extract_pdbs.default.linuxgccrelease ERROR: ambiguous res_num & chain & segid 1

### DISCUSSION THREAD

**Discussion by johnnytam100** [RosettaCommons/rosetta#246] [Developing]
Hi I am trying to extract PDBs with extract_pdbs.default.linuxgccrelease.
However I got an error that completely stopped the extraction:

```
...
core.conformation.Conformation: current variant for 1509 CYD
core.conformation.Conformation: current variant for 1583 CYD
core.conformation.Conformation: current variant for 1509 CYD
core.conformation.Conformation: current variant for 1583 CYD
core.conformation.Conformation: Reverting out-of-date disulfide to thiol type at resid 1530
core.conformation.Conformation: Found disulfide between residues 1631 1634
core.conformation.Conformation: current variant for 1631 CYD
core.conformation.Conformation: current variant for 1634 CYD
core.conformation.Conformation: current variant for 1631 CYD
core.conformation.Conformation: current variant for 1634 CYD
core.conformation.Conformation: Reverting out-of-date disulfide to thiol type at resid 1653

ERROR: ambiguous res_num & chain & segid 1       
ERROR:: Exit from: src/core/pose/full_model_info/FullModelParameters.cc line: 418

[ ERROR ]: Caught exception:


File: src/core/pose/full_model_info/FullModelParameters.cc:418
[ ERROR ] UtilityExitException
ERROR: ambiguous res_num & chain & segid 1       




AN INTERNAL ERROR HAS OCCURED. PLEASE SEE THE CONTENTS OF ROSETTA_CRASH.log FOR DETAILS.
```

Do you have any idea how can I deal with it?
Thank you!

**Comment by roccomoretti** [comment]
The conventional suggestions for issues when reading in silent files is to try the `-in:file:silent_read_through_errors` This will discard any structures with issues. I'm not sure if that will work in this particular case, as I think the error is being thrown from elsewhere.

The alternate approach is to dump the full list of tags in the silent file (Something like `grep 'SCORE:' <silentfilename> | awk '{print $NF}' | grep -v description > tagfile.txt` will do the trick). Then you can try the extraction with the `-in:file:tagfile tagfile.txt` option, which will only try to extract the structures which are listed in the tagfile. -- Simply keep editing the tagfile.txt to remove any structure tags which are causing issues (there should be an indication of which tag is being processed in the tracer output)

If that's all the files, or if there's a particular structure you need which you can't extract, then we may need to drill down onto what's going on. But that would likely require some hands-on-debugging with an example input.
